scripts/zones/Bastok_Markets_[S]/npcs/MillardIM.lua

```diff
@@ -0,0 +1,104 @@
+-----------------------------------
+-- Area: Bastok Markets [S]
+--  NPC: MillardIM
```

Comment:
can you fix her name in both db and script? spaces should change to underscores and , . both get stripped so `Millard_IM`

Reply:
The rest looks ok. Eventually soem of it will get refactored/cleaned up when we get the event params handled better (I did the original for san d'oria's NPC kind of messy)